src/utils/file_utils.py

```diff
@@ -16,10 +21,19 @@ def read_yaml_file(file_path: str) -> Dict:
     return data
 
 
-def get_files_from_folder_path(path: str) -> List:
-    return [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+def get_files_from_folder_path(path: Path) -> Result[List, str]:
+    if check_if_path_exists(path):
+        files = [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
```

Comment:
Alternative if you wanna stick to `Path`:
```suggestion
if path.exists():
    files = [p for p in path.glob("*") if p.is_file()]
```